<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Solver_Setup</class>
 <widget class="QWidget" name="Solver_Setup">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>543</width>
    <height>534</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Garamond</family>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <property name="locale">
   <locale language="English" country="Germany"/>
  </property>
  <widget class="QPushButton" name="Conf_Solver">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>420</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-style: outset;
border-color: grey;
border-width: 3px;
border-radius: 8px;</string>
   </property>
   <property name="text">
    <string>Configure 
 Solver</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="FLowSet">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>50</y>
     <width>62</width>
     <height>22</height>
    </rect>
   </property>
   <property name="locale">
    <locale language="English" country="UnitedKingdom"/>
   </property>
  </widget>
  <widget class="QLabel" name="FLowLabel">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>50</y>
     <width>221</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Lower Frequency</string>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="FHighLabel">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>80</y>
     <width>251</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Upper Frequency</string>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="FHighSet">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>80</y>
     <width>62</width>
     <height>22</height>
    </rect>
   </property>
   <property name="locale">
    <locale language="English" country="UnitedKingdom"/>
   </property>
   <property name="value">
    <double>50.000000000000000</double>
   </property>
  </widget>
  <widget class="QSpinBox" name="FDiscSet">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>110</y>
     <width>42</width>
     <height>22</height>
    </rect>
   </property>
   <property name="value">
    <number>20</number>
   </property>
  </widget>
  <widget class="QLabel" name="FDiscLabel">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>110</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>N Frequencies</string>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="BetaLowSet">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>300</y>
     <width>62</width>
     <height>22</height>
    </rect>
   </property>
   <property name="locale">
    <locale language="English" country="UnitedKingdom"/>
   </property>
   <property name="maximum">
    <double>3600000.000000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="BetaDiscLabel">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>360</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>N Angles</string>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QSpinBox" name="BetaDiscSet">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>360</y>
     <width>42</width>
     <height>22</height>
    </rect>
   </property>
   <property name="value">
    <number>1</number>
   </property>
  </widget>
  <widget class="QLabel" name="BetaHighLabel">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>330</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Upper Angle</string>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="BetaHighSet">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>330</y>
     <width>62</width>
     <height>22</height>
    </rect>
   </property>
   <property name="locale">
    <locale language="English" country="UnitedKingdom"/>
   </property>
   <property name="maximum">
    <double>3600000.000000000000000</double>
   </property>
   <property name="value">
    <double>180.000000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="BetaLowLabel">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>300</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Lower Angle</string>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_5">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>140</y>
     <width>221</width>
     <height>101</height>
    </rect>
   </property>
   <property name="title">
    <string>Frequency Spacing</string>
   </property>
   <widget class="QRadioButton" name="FDiscCos2Button">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>231</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Cosine (resolved upper)</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="FDiscLinButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>251</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Linear Discretisation</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="FDiscCos1Button">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>231</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Cosine (resolved lower)</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_6">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>221</width>
     <height>81</height>
    </rect>
   </property>
   <property name="title">
    <string>Water depth</string>
   </property>
   <widget class="QRadioButton" name="FinDepthButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>171</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Finite Depth [m]</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="InfDepthButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>161</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Infinite Depth</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="FinDepthSet">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>50</y>
      <width>62</width>
      <height>22</height>
     </rect>
    </property>
    <property name="locale">
     <locale language="English" country="UnitedKingdom"/>
    </property>
   </widget>
  </widget>
  <widget class="QDoubleSpinBox" name="GravitySpin">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>140</y>
     <width>62</width>
     <height>22</height>
    </rect>
   </property>
   <property name="locale">
    <locale language="English" country="UnitedKingdom"/>
   </property>
   <property name="maximum">
    <double>10000.000000000000000</double>
   </property>
   <property name="value">
    <double>9.810000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="DensitySpin">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>200</y>
     <width>62</width>
     <height>22</height>
    </rect>
   </property>
   <property name="locale">
    <locale language="English" country="UnitedKingdom"/>
   </property>
   <property name="maximum">
    <double>10000.000000000000000</double>
   </property>
   <property name="value">
    <double>1025.000000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>211</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Gravity [m/s^2]</string>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>170</y>
     <width>261</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Water density [kg/m^3]</string>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="FileNameEdit">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>450</y>
     <width>231</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>BEMUse_Output</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="OutputNameLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>450</y>
     <width>181</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Output Filename</string>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>261</width>
     <height>231</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-style: outset;
border-color: white;
border-width: 6px;
border-radius: 24px;</string>
   </property>
   <property name="title">
    <string>Environmental Options</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>20</y>
     <width>241</width>
     <height>231</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-style: outset;
border-color: white;
border-width: 6px;
border-radius: 24px;</string>
   </property>
   <property name="title">
    <string>Frequency Range</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_4">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>260</y>
     <width>241</width>
     <height>151</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-style: outset;
border-color: white;
border-width: 6px;
border-radius: 24px;</string>
   </property>
   <property name="title">
    <string>Wave angle Range</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>260</y>
     <width>261</width>
     <height>151</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-style: outset;
border-color: white;
border-width: 6px;
border-radius: 24px;</string>
   </property>
   <property name="title">
    <string>Solver Options</string>
   </property>
   <widget class="QCheckBox" name="IRFToggle">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>291</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Garamond</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border: none;</string>
    </property>
    <property name="text">
     <string>Irregular Frequency 
Removal</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="KochinDiscSet">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>91</y>
      <width>42</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:none;</string>
    </property>
    <property name="value">
     <number>36</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>90</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Garamond</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border:none;</string>
    </property>
    <property name="text">
     <string>N Angles Kochin</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <zorder>groupBox</zorder>
  <zorder>groupBox_4</zorder>
  <zorder>groupBox_3</zorder>
  <zorder>groupBox_6</zorder>
  <zorder>Conf_Solver</zorder>
  <zorder>FLowSet</zorder>
  <zorder>FLowLabel</zorder>
  <zorder>FHighLabel</zorder>
  <zorder>FHighSet</zorder>
  <zorder>FDiscSet</zorder>
  <zorder>FDiscLabel</zorder>
  <zorder>BetaLowSet</zorder>
  <zorder>BetaDiscLabel</zorder>
  <zorder>BetaDiscSet</zorder>
  <zorder>BetaHighLabel</zorder>
  <zorder>BetaHighSet</zorder>
  <zorder>BetaLowLabel</zorder>
  <zorder>groupBox_5</zorder>
  <zorder>GravitySpin</zorder>
  <zorder>DensitySpin</zorder>
  <zorder>label</zorder>
  <zorder>label_2</zorder>
  <zorder>FileNameEdit</zorder>
  <zorder>OutputNameLabel</zorder>
  <zorder>groupBox_2</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
